17.0.Dockerfile

```diff
@@ -0,0 +1,232 @@
+FROM python:3.8-slim-bullseye AS base
```

Comment:
Odoo 17 need minimum  PY 3.0 according this commit
https://github.com/odoo/odoo/commit/163fb3ae618bf993e60f8c1276e09a5b85565fa5
Do you sure this working with 3.8?

Reply:
I see, we will need to make it work with P3.10. I was delaying such step, as it was conflicting for 16 (and still not working).

Reply:
I tested into commit https://github.com/celm1990/doodba/commit/888f30b6fca66b54daba167a6f3917d89e68f324 and it's working now